docs/datadog_monitor.md

```diff
@@ -130,3 +142,4 @@ kubectl logs <my-datadog-operator-pod-name>
 [5]: https://app.datadoghq.com/account/settings#api
 [6]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog-operator/values.yaml
 [7]: https://app.datadoghq.com/monitors/manage?q=tag%3A"generated%3Akubernetes"
+[8]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog-operator/values.yaml#L147-L156
```

Comment:
Since line numbers will change on `main`, and we want the doc to be timeless, perhaps we can remove this reference and just re-use [6]

Reply:
Hi @celenechang. Apologies for the delay.  Just wanted to see if this PR was good to go or already released by another PR?

Reply:
Hi! I'm sorry too. This looks good to me. Just an FYI, there are more refined configuration options coming with the next Operator version, but we can update the doc again after it's released